vllm/model_executor/models/registry.py

```diff
@@ -88,7 +88,6 @@
     "JAISLMHeadModel": ("jais", "JAISLMHeadModel"),
     "JambaForCausalLM": ("jamba", "JambaForCausalLM"),
     "LlamaForCausalLM": ("llama", "LlamaForCausalLM"),
-    "Llama4ForCausalLM": ("llama4", "Llama4ForCausalLM"),  # noqa: E501
```

Comment:
Looks like adding this leads to CI errors https://github.com/vllm-project/vllm/pull/19580